<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DropboxBrowser</class>
 <widget class="QDialog" name="DropboxBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dropbox File Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTreeView" name="treeView">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QListView" name="listView">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="cmdUpload">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tosec.qrc">
         <normaloff>:/icon/player/blue/images/player/blue/forward.ico</normaloff>:/icon/player/blue/images/player/blue/forward.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdDownload">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../tosec.qrc">
         <normaloff>:/icon/player/blue/images/player/blue/backward.ico</normaloff>:/icon/player/blue/images/player/blue/backward.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lblToken">
         <property name="text">
          <string>Token</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtToken"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblPublisher">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>19</pointsize>
        </font>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="cmdGetFiles">
       <property name="text">
        <string>List Dropbox Files</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblDir">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Dir:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtDir"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QListWidget" name="listDropbox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="cmdMakedir">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>mkdir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdRemove">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../tosec.qrc"/>
 </resources>
 <connections/>
</ui>
